Sources/AsyncConnection.swift

```diff
@@ -3,7 +3,7 @@ public protocol AsyncConnection: AsyncStream {
 }
 
 extension AsyncConnection {
-    public func open(result: (Void throws -> AsyncConnection) -> Void) throws {
+    public func open(_ result: (Void throws -> AsyncConnection) -> Void) throws {
```

Comment:
@dreymonde I agree. I wasn't arguing against the label, just the name of the typealias if we were to use one. I think the label is fine. :)